C++/lame.cpp

```diff
@@ -0,0 +1,55 @@
+/*  This code can not be expected form even a 5 year old 
+    who has started to learn programming. I am proud of it though XD
+*/
+#include<iostream>
+using namespace std;
+class LameFizzBuzz {
+    int lastCounter, currentCounter, max;
+public:
+    //Constructor to initiate the counter variables
+    LameFizzBuzz(int iValue, int limit) {
+        lastCounter = iValue;
+        max = limit;
+        currentCounter = lastCounter + 1;
+    }
+    //Beware: the following function conatins infinitely many return statements.

```

Comment:
Three returns?
![image](https://user-images.githubusercontent.com/2089820/31319274-da9208ac-ac25-11e7-8159-84fee4f273d0.png)

Reply:
Yes Sir. I am very excited to be a part of this madness.